libandroid-navigation/src/test/java/com/mapbox/services/android/navigation/v5/navigation/MapboxOfflineRouterTest.kt

```diff
@@ -2,65 +2,66 @@ package com.mapbox.services.android.navigation.v5.navigation
 
 import com.mapbox.geojson.BoundingBox
 import com.mapbox.geojson.Point
+import io.mockk.mockk
+import io.mockk.verify
 import org.junit.Test
-import org.mockito.ArgumentMatchers.eq
-import org.mockito.Mockito.mock
-import org.mockito.Mockito.verify
 
 class MapboxOfflineRouterTest {
 
     @Test
     fun initializeOfflineData_filePathIncludesVersion() {
         val tilePath = "/some/path/"
-        val offlineNavigator = mock(OfflineNavigator::class.java)
-        val callback = mock(OnOfflineTilesConfiguredCallback::class.java)
+        val offlineNavigator = mockk<OfflineNavigator>(relaxUnitFun = true)
```

Comment:
Could you describe what `relaxUnitFun` does? What are the main benefits of using `mockk` instead of `mockito`?

Reply:
`Relaxed` mock obj returns some default value when return value unspecified by `Mock` framework. `relaxUnitFun` means that func should return `Unit` instead throw exception. Better to read about it [here](https://mockk.io/#relaxed-mock) 
`Mockk` makes code more kotin-style (`blocks`, `lambdas support`, `verify/every blocks`). It has 'spy' feature, using on you can change behavior of testing object. Here's a lot of great features, please have a look at [docs](https://mockk.io/)

Reply:
Ohh I see, something similar to Mockito's [`RETURNS_DEEP_STUBS`](https://static.javadoc.io/org.mockito/mockito-core/3.1.0/org/mockito/Mockito.html#RETURNS_DEEP_STUBS). Thanks for clarifying and for describing other interesting features!

Reply:
welcome)